plugins/salesforce/komand_salesforce/util/api.py

```diff
@@ -368,11 +368,18 @@ def get_error(response: str) -> Tuple[str, str, Any]:
                     401,
                 )
             if error_desc:
-                return (
-                    f"Salesforce error: '{error_desc}'",
-                    PluginException.assistances[PluginException.Preset.UNKNOWN],
-                    "",
-                )
+                if "retry your request" in error_desc:
```

Comment:
We still sort of have an 'unknown' for the assistance, so we could simplify to apply a status code on this else and reduce duplicate lines?

```
if "retry your request" in error_desc:
  status_code = 500
else:
  status_code = ""
  <new_logger> 

return (.....)  
````